src/components/SlateEditor/plugins/embed/EditImage.tsx

```diff
@@ -135,6 +145,9 @@ const EditImage = ({ embed, saveEmbedUpdates, setEditModus, language }: Props) =
               onChange={onChange}
               onAbort={onAbort}
               onSave={onSave}
+              isDecorative={state.isDecorative}
+              handleCheck={handleCheck}
+              allowDecorative={allowDecorative !== false}
```

Comment:
Dette ser ut som en forvansking. Om du må sjekke verdien på en boolsk har du gjort noe feil.
```suggestion
              allowDecorative={allowDecorative !== false}
```